daily_post.py

```diff
@@ -26,8 +26,10 @@ def get_weather():
     doc = resp.content
     soup_doc = BeautifulSoup(doc, 'html.parser')
     weather_info = soup_doc.find_all('table')[1]
-    forecast = weather_info.contents[0].text
-    reddit_comment = '* ' + str.replace(forecast, '\n\n',"\n* ")[:-3]
+    forecast = str(weather_info.contents[0]).split("<br/>\n<br/>")
+    top_three = '<br/>\n<br/>'.join(str(x) for x in forecast[1:3])
```

Comment:
The major problem here is that on occasion, the forecast will have a "special update" at the top where the just arbitrarily throw it in :| It would be easy to catch if they used a constant tag or something, but they don't.

So in the event of a special update, this would return just the special update, and end up cutting off part of the day's weather.

Reply:
Any chance you have a sample of the api response when it includes a special update?

Reply:
I don't have a specific BS4 response, unfortunately, but here are a few posts that have been caused from it:

https://www.reddit.com/r/SeattleWA/comments/7e7dlw/seattle_reddit_community_open_chat_monday/

https://www.reddit.com/r/SeattleWA/comments/7epce5/seattle_reddit_community_open_chat_wednesday/

https://www.reddit.com/r/SeattleWA/comments/7fdxkh/seattle_reddit_community_open_chat_saturday/

I'm wondering if we can at least get a list of known curveballs like this and just add a really dumb if block for formatting it if it comes in haha

Reply:
I managed to find some live versions.
[Example-Air-Stagnation-Advisory.txt](https://github.com/r-seattle-wa/SeattleRedditBot/files/1529719/Example-Air-Stagnation-Advisory.txt)
[Example-Hazardous-Weather-Outlook.txt](https://github.com/r-seattle-wa/SeattleRedditBot/files/1529720/Example-Hazardous-Weather-Outlook.txt)

I'll see if I can find if there are any examples. It's wrapped in a div and anchor tag so we can target it for removal if we want or scrap the link if we want.

Reply:
[Example-Double-Advisories.txt](https://github.com/r-seattle-wa/SeattleRedditBot/files/1529728/Example-Double-Advisories.txt)

Found one. I'll update the splicer section to handle these as their own thing.